### PR TITLE
sdk_automl_image_object_detection_batch

### PR DESCRIPTION
**Pull request by udaypunna**
**REQUIRED:** Add a summary of your PR here, typically including why the change is needed and what was changed. Include any design alternatives for discussion purposes.

<br>
boiler plate changes.
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [X] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [X] Follow the style and grammar rules outlined in the above notebook template.
- [X] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [X] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [X] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [X] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb [hunk 1]
```diff
@@ -8,7 +8,7 @@
       },
```
Comment (ivanmkc) [inline notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb:8]: <pre class="ql-syntax" spellcheck="false">Line #2.        display_name="Salads" + "_" + "unique",
</pre><p>Just write: "salads_unique"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1240/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Comment (ivanmkc) [inline notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb:8]: <pre class="ql-syntax" spellcheck="false">Line #2.        display_name="salads_" + "unique",
</pre><p>Same here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1240/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Comment (ivanmkc) [inline notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb:8]: <pre class="ql-syntax" spellcheck="false">Line #3.        model_display_name="salads_" + "unique",
</pre><p>Same.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1240/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Comment (ivanmkc) [inline notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb:8]: <pre class="ql-syntax" spellcheck="false">Line #11.    print(model)
</pre><p>Remove.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1240/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Comment (ivanmkc) [inline notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb:8]: <pre class="ql-syntax" spellcheck="false">Line #2.    models = aiplatform.Model.list(filter="display_name=salads_" + "unique")
</pre><p>Same.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1240/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Comment (ivanmkc) [inline notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb:8]: <pre class="ql-syntax" spellcheck="false">Line #2.        job_display_name="salads_" + "unique",
</pre><p>Same.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1240/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

[the remark this answers is not in the document]
Comment (udaypunna) [reply notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb:8]: <p>removed $user_flag</p>

Comment (udaypunna) [reply notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb:8]: <p>renamed to salads_unique</p>

**Pull request merged**